src/libs/OptionsListUtils.js

```diff
@@ -398,7 +398,11 @@ function getLastMessageTextForReport(report) {
                 reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE &&
                 ReportActionUtils.isMoneyRequestAction(reportAction),
         );
-        lastMessageTextFromReport = ReportUtils.getReportPreviewMessage(iouReport, lastIOUMoneyReport, true);
+        // Weather parent report is a 1:1 report
+        const isChatReport = ReportUtils.isChatReport(report);
```

Comment:
Should we just add a condition here rather than passing the param all the way to `isSettled`?

Like if the iouReport has `isWaitingOnBankAccount` set and the current report is a DM? And let's create a separate function for showing settled message?

```
        // A settled report preview message can come in three formats "paid ... elsewhere" or "paid ... with Expensify"
        let translatePhraseKey = 'iou.paidElsewhereWithAmount';
        if (
            _.contains([CONST.IOU.PAYMENT_TYPE.VBBA, CONST.IOU.PAYMENT_TYPE.EXPENSIFY], lodashGet(reportAction, 'originalMessage.paymentType')) ||
            reportActionMessage.match(/ (with Expensify|using Expensify)$/)
        ) {
            translatePhraseKey = 'iou.paidWithExpensifyWithAmount';
        }
        return Localize.translateLocal(translatePhraseKey, {amount: formattedAmount, payer: payerName});
```

Reply:
@situchan ^ because I saw your comment regarding new code in `isSettled` method.

Reply:
@techievivek won't this be duplicate of same logic again? Please check I have updated the condition to not to include variables inside `isSettled` function.